kotlinlib/src/mill/kotlinlib/android/ComposeRenderer.scala

```diff
@@ -0,0 +1,57 @@
+package mill.kotlinlib.android
+
+import mill.api.internal
+
+private[android] object ComposeRenderer {
```

Comment:
Are these data structures written based on any external schema? If so we should add a link in the comment

Reply:
thanks! yes, added link!